bin/cf-blue-green-travis

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -e
+
+wget http://go-cli.s3-website-us-east-1.amazonaws.com/releases/v6.12.1/cf-cli_amd64.deb -qO temp.deb && sudo dpkg -i temp.deb
```

Comment:
Where did this URL come from? We should add a URL as a comment, so we know where to look for updates.